vars/os_RedHat_8.yml

```diff
@@ -1,6 +1,6 @@
 ---
 docker_unit_after: "multi-user.target"
-docker_storage_driver: overlay2
+docker_storage_driver: overlay
```

Comment:
I am guessing RHEL 8 doesn't have overlay2, which is a little surprising.

Reply:
I've currently a problem with overlay2. And I'm suspecting that one is an alias of the other on rhel8, just for for investigating